app/js/factories/UserAuth.js

```diff
@@ -25,14 +25,22 @@ function UserAuth(UserResourceService, $cookies,$log) {
     	return new Promise((resolve)=>{
           
         if(userSlug || this._getUserFromStorage()){
-
           this._authenticateUser(userSlug)
-                .then(authedCookieUser=>{
-                  resolve(authedCookieUser);
+            .catch(error => {
+              $log.log(`login:: login failed with slug ${userSlug}, creating new user`);
```

Comment:
Oh interesting, I'm not familiar with this `::` syntax - does it do anything in particular or just print them out?